test-e2e/media-server.js

```diff
@@ -19,6 +20,27 @@ const projectMigrationsFolder = new URL('../drizzle/project', import.meta.url)
 const clientMigrationsFolder = new URL('../drizzle/client', import.meta.url)
   .pathname
 
+test('start/stop lifecycle', async (t) => {
+  const manager = new MapeoManager({
+    rootKey: KeyManager.generateRootKey(),
+    projectMigrationsFolder,
+    clientMigrationsFolder,
+    dbFolder: ':memory:',
+    coreStorage: () => new RAM(),
+  })
+
+  await manager.startMediaServer()
```

Comment:
Check it's started: get a url, check it works.

Reply:
Don't need to put data there, just returning 404 is enough to know it's started.